### ISSUE TITLE
arXiv:1904.07200

### ISSUE THREAD

**Issue by refraction-ray** [quclub/Paper-reading#62]
* [Link](https://arxiv.org/pdf/1904.07200.pdf)
* Title: A Discussion on Solving Partial Differential Equations using Neural Networks
* Keywords (optional):  
* Authors (optional): 
* Reason (optional): 
* Summary (optional): Discretization, plug boundary condition into loss function. Possion solution reproduced
* Ideas (optional):  I will maintain a list of work at the interplay between PDE and DL with particular focus on use NN ansatz to solve PDEs.

**Comment by refraction-ray** [comment]
## Relevant work
* [DGM: A deep learning algorithm for solving partial differential equations](https://arxiv.org/pdf/1708.07469.pdf) Just every person chooses to do: meshfree, MC sample batch, add boundary condition to the loss, nothing new
* [Artiial Neural Networks for Solving Ordinary and Partial Differential Equations](https://arxiv.org/pdf/physics/9705023.pdf)
* [Accelerating Eulerian Fluid Simulation With Convolutional Networks](https://arxiv.org/pdf/1607.03597.pdf)
* [Deep Hidden Physics Models: Deep Learning of Nonlinear Partial Differential Equations](https://arxiv.org/pdf/1801.06637.pdf)
* [Physics Informed Deep Learning (Part I): Data-driven Solutions of Nonlinear Partial Differential Equations](https://arxiv.org/pdf/1711.10561.pdf)
